### ISSUE TITLE
Ignoring empty lines when creating MarkDown documentation

### ISSUE THREAD

**Issue by horribleCodes** [365businessdev/vscode-alxmldocumentation#53]
I use a definition list in the summary of my Pages, but when I generate the Markdown file it always ends up skipping a linebreak, breaking the format.

For example, the comment might look like this:
```{r tidy=FALSE}
/// <summary>
/// Lists all currently active sessions.
/// 
/// **Type**
/// : List\
/// **Source**
/// : Active Sessions
/// </summary>
```
But when generating the .md, the third line gets removed:

```{r tidy=FALSE}
Lists all currently active sessions.
**Type**
: List\
**Source**
: Active Sessions
```

Which, in the VS Code Preview, ends up looking like this:

![image](https://user-images.githubusercontent.com/7748313/114677121-494dc780-9d0a-11eb-834a-b9d04ef1a042.png)

The above code is displayed correctly at least on github, but if this gets primarily viewed in VS Code, I'd like it to work there as well without having to manually fix the markdown document.

**Comment by sirhc101** [comment]
Thank you for reporting. We've already fixed this. With the new documentation export it looks this way:
![image](https://user-images.githubusercontent.com/39621236/114715132-0b18ce00-9d33-11eb-9596-fa8a3ad125a1.png)

We're currently in final tests and plan to release the new version at friday.

**Comment by horribleCodes** [comment]
Great, thanks for the quick response.

**Comment by sirhc101** [comment]
Issue will be closed after release of v1.1
